backend/src/zango/apps/appauth/models.py

```diff
@@ -171,7 +171,7 @@ def create_user(
                             app_user.is_active = True
 
                         if not force_password_reset:
-                            old_password_obj = OldPasswords.objects.create(user=user)
+                            old_password_obj = OldPasswords.objects.create(user=app_user)
                             old_password_obj.setPasswords(user.password)
```

Comment:
can you please change this also to `app_user.password` ?

Reply:
I removed line 254 and fixed the call for the password but I'm not fully sure how to solve the problem where it asks again I removed the extra if statement because it feels unnecessary the more I thought about it

Reply:
for some reason, it pushed to my school account to Im gonna resubmit through the correct one